test/behavior/tendrl/login.feature

```diff
@@ -12,29 +12,12 @@ Feature: LOGIN
       And User deletes what is in the "password" field
      Then The text "password is required" should appear under the "password" box
 
-  Scenario: Click login with empty fields
-    Given User is at the Tendrl login page
-     When User clicks the Login button
-     Then The “Please fill out this field" popup should appear under the "username" box
-
-  Scenario: Click login button with username but no password
-    Given User is at the Tendrl login page
-     When User types something in the "username" field
-      And User clicks the Login button
-     Then The “Please fill out this field" popup should appear under the "password" box
-
-  Scenario: Click login button with password but no username
-    Given User is at the Tendrl login page
-     When User types something in the "password" field
-      And User clicks the Login button
-     Then The “Please fill out this field" popup should appear under the "username" box
-
   Scenario: Incorrect Username and Password
     Given User is at the Tendrl login page
      When User types in an incorrect "username"
```

Comment:
difference between incorrect and something in 
Scenario: Incorrect Username and Password 
Scenario: Tendrl API is not reachable

why can't we write something more generic for both

Reply:
We could ax the "something" case and just use the incorrect? The input for the something case does not matter so it isn't explicitly needed

Reply:
I think we can use incorrect for both.